src/main/java/com/iota/iri/controllers/TransactionViewModel.java

```diff
@@ -19,14 +19,20 @@
 
     private final Transaction transaction;
 
-    /** Length of a transaction object in trytes */
```

Comment:
you haven't changed any code in this file - only styling.
there are 59 files to review with a lot of changes, please try to disable auto-formatting/styling so there is less to review.